src/main/java/io/supertokens/authRecipe/AuthRecipe.java

```diff
@@ -97,15 +110,16 @@ public static UserPaginationContainer getUsers(Main main,
                                                    @Nullable RECIPE_ID[] includeRecipeIds)
             throws StorageQueryException, UserPaginationToken.InvalidTokenException {
         try {
-            return getUsers(new TenantIdentifier(null, null, null), main, limit, timeJoinedOrder, paginationToken,
+            Storage storage = StorageLayer.getStorage(main);
+            return getUsers(new TenantIdentifier(null, null, null, storage), limit, timeJoinedOrder, paginationToken,
                     includeRecipeIds);
         } catch (TenantOrAppNotFoundException e) {
             throw new IllegalStateException(e);
         }
     }
 
-    public static void deleteUser(TenantIdentifier tenantIdentifier, Main main, String userId)
-            throws StorageQueryException, TenantOrAppNotFoundException, BadPermissionException {
+    public static void deleteUser(AppIdentifier appIdentifier, String userId, UserIdMapping userIdMapping)
+            throws StorageQueryException, BadPermissionException {
```

Comment:
BadPermissionException ->  it's not actually thrown from this function? Please be more mindful of these types of changes

Reply:
Done